application/forms/Config/UserGroup/UserGroupBackendForm.php

```diff
@@ -190,4 +214,91 @@ public function onRequest()
             $this->populate($data);
         }
     }
+
+    /**
+     * Run the configured backend's inspection checks and show the result, if necessary
+     *
+     * This will only run any validation if the user pushed the 'backend_validation' button.
+     *
+     * @param   array   $formData
+     *
+     * @return  bool
+     */
+    public function isValidPartial(array $formData)
+    {
+        if (! parent::isValidPartial($formData)) {
+            return false;
+        }
+
+        if ($this->getElement('backend_validation')->isChecked() && parent::isValid($formData)) {
```

Comment:
Once the call to `Form::isValidPartial` is gone, this needs to change. (e.g. with `isset()`)